addon/ng2/blueprints/ng2/files/src/app/__name__.e2e.ts

```diff
@@ -0,0 +1,9 @@
+describe('<%= jsComponentName %> App', function() {
+
+  // #docregion redirect
+  it('should display message saying app works', function() {
+    browser.get('/');
+    expect(element(by.css('<%= htmlComponentName %>-app p')).getText())
```

Comment:
[toHaveText](https://github.com/angular/angular/blob/master/modules/angular2/src/testing/matchers.ts#L9) is the only one that seems applicable.

Trying to import anything from `import {} from 'angular2/testing';` seems to give me  `[launcher] Error: ReferenceError: System is not defined` in protractor. 

This seems to be because adding imports makes the generated files use `System`. Will have to look into how to make that work on protractor.